Xamarin.Forms.Pages dependes on modernhttpclient?

### Description

When targeting `netstandard,` I get this build warning:

> warning NU1701: Package 'modernhttpclient 2.4.2' was restored using '.NETPortable,Version=v0.0,Profile=Profile259, .NETFramework,Version=v4.6.1' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.

This is seemingly because Xamarin.Forms.Pages has this dependency on ModernHttpClient? Why is this dependency needed? Does Xamarin.Forms.Pages make web calls? The nuget package isn't built for netstandard yet.

However this is a fork that is: https://github.com/alexrainman/ModernHttpClient

Can you consider dropping this dependency, or switching to netstadnard compataible one.

In my experience the application should be approvide the HttpClient handler, so it should be an option for the application to configure this, not something that xamarin enforces?

--- a/.nuspec/Xamarin.Forms.Pages.Azure.nuspec
+++ b/.nuspec/Xamarin.Forms.Pages.Azure.nuspec
@@ -15,7 +15,6 @@
       <group>
         <dependency id="Xamarin.Forms.Pages" version="$version$"/>
         <dependency id="Newtonsoft.Json" version="10.0.3"/>
-        <dependency id="modernhttpclient" version="2.4.2"/>
         <dependency id="Microsoft.Azure.Mobile.Client" version="4.0.2" />
       </group>
     </dependencies>

--- a/.nuspec/Xamarin.Forms.Pages.nuspec
+++ b/.nuspec/Xamarin.Forms.Pages.nuspec
@@ -15,7 +15,6 @@
       <group>
         <dependency id="Xamarin.Forms" version="$version$"/>
         <dependency id="Newtonsoft.Json" version="10.0.3"/>
-        <dependency id="modernhttpclient" version="2.4.2"/>
       </group>
     </dependencies>
   </metadata>

